GPflowOpt/acquisition.py

```diff
@@ -168,47 +170,63 @@ def build_acquisition(self, Xcand):
         return normal.cdf(tf.constant(0.0, dtype=float_type), name=self.__class__.__name__)
 
 
-class AcquisitionBinaryOperator(Acquisition):
-    """
-    Base class for implementing binary operators for acquisition functions, for defining joint acquisition functions
-    """
-
-    def __init__(self, lhs, rhs, oper):
-        super(AcquisitionBinaryOperator, self).__init__()
-        assert isinstance(lhs, Acquisition)
-        assert isinstance(rhs, Acquisition)
-        self.lhs = lhs
-        self.rhs = rhs
+class AcquisitionAggregationOperator(Acquisition):
```

Comment:
Actually this is quite a long name. anything shorter? although we are living the world of autocompletion, something like AcquisitionAggregator might be slightly easier to remember?